docs/docs/data-fetching.md

```diff
@@ -88,8 +89,11 @@ exports.sourceNodes = async ({
 }
 ```
 
+It is important to note that node fields can indeed be arbitrary. The `nameWithOwner` and `url` fields from above are examples of this.
+
 This node created manually could be retrieved with a query like this:
 
+
```

Comment:
```suggestion
```

It looks like this line is likely what's causing the linting tests to fail, could you remove the empty new line as well?